security/subsystem/src/main/java/org/jboss/as/security/SecurityDomainAdd.java

```diff
@@ -156,6 +165,7 @@ public void launchServices(OperationContext context, String securityDomain, Mode
         final ApplicationPolicy applicationPolicy = createApplicationPolicy(context, securityDomain, model);
         final JSSESecurityDomain jsseSecurityDomain = createJSSESecurityDomain(context, securityDomain, model);
         final String cacheType = getAuthenticationCacheType(model);
+        final boolean exportElytronRealm = exportElytronRealm(model);
```

Comment:
This should be

```
final boolean exportElytronRealm = SOME_ATTRIBUTE_DEFINITION.resolveModelAttribute(context, model).asBoolean();
```
SOME_ATTRIBUTE_DEFINITION should have a default value of false.

Your exportElytronRealm method is a non-standard impl of the same thing, and those tend to become bugs as the code evolves.